### DISCUSSION TITLE
Backup Casa OS

### DISCUSSION THREAD

**Discussion by lucifer78-dev** [IceWhaleTech/CasaOS#1415] [Q&A]
Hi. I am new to Casa OS and I have installed the system running on a PC with debian.
I am using Duplicati to backup my data.

My question is: Is there any way to backup the Casa OS system files for recovery?
In case of a breakdown, how do I recover the Casa OS installation?

 Thanks you in advance.

**Comment by favoyang** [comment]
Backup both `/var/lib/casaos` and `/DATA`.

**Reply by BitFog** [reply]
hey, if I have to recover casa os to a fresh install of ubuntu, will copying these two files work, do I have to paste these files before or after I reinstall casa os ? 
Thanks in advance

**Reply by isyuricunha** [reply]
> hey, if I have to recover casa os to a fresh install of ubuntu, will copying these two files work, do I have to paste these files before or after I reinstall casa os ? 
> Thanks in advance

Install CasaOS, but don't access it, past the files, and access normally. 

Sometimes u find problems with permissions read/write, sometimes no. 

IMO, CasaOS need one feature/tool to backup and import backup's.
